src/index.html

```diff
@@ -4,19 +4,25 @@
     <meta charset="UTF-8" />
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, initial-scale=1.0"
     />
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Stop watch</title>
+    <title>Stopwatch</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The link tag should refer to a CSS file, not an SCSS file. SCSS is a preprocessor and needs to be compiled into CSS for the browser to understand it. You should change the href attribute to point to the compiled CSS file.